### PR TITLE
CMR-4173: Fixed test and mdb search code for services.

### PR DESCRIPTION
**Pull request by indiejames**
Needed to support services.

### CODE REVIEW

File: metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj [hunk 1]
```diff
@@ -423,40 +423,47 @@
...
+               (set (->> (util/find-latest-concepts :service params)
+                         :concepts
+                         (map #(dissoc % :provider-id :revision-date :transaction-id)))))
+            "with metadata search by concept-id"
+            [serv1] {:concept-id (:concept-id serv1)}
+
+            "with metadata search by native-id"
+            [serv2] {:native-id (:native-id serv2)}
+
+            "exclude metadata serach by concept-id"
```
Comment (chris-durbin) [inline metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj]: search typo

File: metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj [hunk 2]
```diff
@@ -423,40 +423,47 @@
...
-;                             :concepts
-;                             count)]
-;         (is (= 5 num-of-servs))))))
-;
-; (deftest find-services-with-invalid-parameters
-;   (testing "extra parameters"
-;     (is (= {:status 400
-;             :errors ["Finding concept type [service] with parameters [short-name] is not supported."]}
-;            (util/find-concepts :service {:short-name "SN1"})))))
+(deftest find-services
```
Comment (ygliuvt) [inline metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj]: NIT: For completeness, add a test where the search parameter matched no service. I know we are not doing this for any of the existing concept types. :)

File: metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj (hunk 1 above)
Comment (indiejames) [inline metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj]: Fixed.

File: metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj (hunk 2 above)
Comment (indiejames) [inline metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj]: Done.

**Pull request merged**